ebpf-io/ebpf.io-website
# feat: adding learning ebpf book and swag shop link

**This PR:**
- Adding "Learning eBPF" book to Get Started page;
- Adding "Swag shop" link to footer;
- Fixing some props validation issues.

**Steps to test**

- Open the [get started page](https://deploy-preview-382--ebpfio.netlify.app/get-started/)
- Confirm that everything looks and works as expected
